test/test_block_store_service.py

```diff
@@ -19,7 +19,7 @@ def test_crud_volume(self):
         Create a new volume, check whether the expected values are set,
         and delete it
         """
-        name = "CBUnitTestCreateVol-{0}".format(uuid.uuid4())
+        name = "CBUnitTestCreateVol-{0}".format(uuid.uuid4().hex[:6])
```

Comment:
Azure resource names have [length limitation.](https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions)  We are using 6 chars from uuid to make it consistent.